### DISCUSSION TITLE
Connection Failed: br-connection-profile-unavailable

### DISCUSSION THREAD

**Discussion by MuteSTEVE** [blueman-project/blueman#2149] [Q&A]
blueman:
BlueZ:
Distribution:
Desktop environment:

* [ ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

<!--

ℹ Please use some service like [Pastebin](http://pastebin.com/) or [GitHub Gist](https://gist.github.com/) to post your logs and keep the thread clean and readable. Make sure to describe the exact steps you took when producing it.

->

**Comment by infirit** [comment]
For most users you haven't installed a soundserver like pulseaudio. But as you provide very little information so not much else to say I guess.
